aspnetcore/blazor/components/index.md

```diff
@@ -1478,9 +1476,6 @@ In the preceding code, the CSS selector, `#app`, indicates that the `App` compon
 <div id="app">...</app>
 ```
 
-<!-- UPDATE 8.0 Need to check on this: Is this still valid in a 
-     BW app, and is the render-mode still set in this way? -->
-
 MVC and Razor Pages apps can also use the [Component Tag Helper](xref:Microsoft.AspNetCore.Mvc.TagHelpers.ComponentTagHelper) to register statically-rendered Blazor WebAssembly root components:
```

Comment:
Yeah, I'm not sure why this specifically refers to using the component tag helper with statically rendered Blazor WebAssembly components. The component tag helper supports the interactive server and WebAssembly render modes. It doesn't support Auto.

Reply:
Let's discuss this further here. This whole section is specifically about rendering static root components. I thought it was correct in that context.